lib/companies_web/views/company_view.ex

```diff
@@ -23,7 +23,8 @@ defmodule CompaniesWeb.CompanyView do
   end
 
   def markdown_format(markdown) do
-    markdown
+    Earmark.as_html!(markdown)
+    |> Phoenix.HTML.raw()
```

Comment:
I've been trying to keep the codebase as closely aligned to the community styleguide as possible: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines

```suggestion
    markdown
    |> Earmark.as_html!()
    |> Phoenix.HTML.raw()
```